summariser/src/query.rs

```diff
@@ -179,6 +184,179 @@ pub async fn zome_call_error_count(
     }
 }
 
+/// Query [`DataFrame`] for any given wind-tunnel metric
+///
+/// Query will filter by time if `summary.run_duration` has been set.
+/// Query may also filter for a specific a tag value if provided.
+pub async fn query_metrics(
+    client: &influxdb::Client,
+    summary: &RunSummary,
+    measurement: &str,
+    columns: &[&str],
+    filter_by_tag: Option<(&str, &str)>,
+) -> anyhow::Result<DataFrame> {
+    let mut cols = columns.join(", ");
+    if !cols.is_empty() {
+        cols = format!(", {}", cols);
```

Comment:
Is this to add a leading `, ` to the string? I think you can use [insert_str](https://doc.rust-lang.org/std/string/struct.String.html#method.insert_str) instead of allocating a new string.

```suggestion
        cols.insert_str(0, ", ");
```